src/GLdispatch/vnd-glapi/entry_files.mk

```diff
@@ -3,35 +3,35 @@
 # The conditionals here are all set from the configure script.
 
 if GLDISPATCH_TYPE_X86_TLS
-MAPI_GLDISPATCH_ENTRY_FILES = $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_tls.c
+MAPI_GLDISPATCH_ENTRY_FILES = mapi/entry_x86_tls.c
 endif
 
 if GLDISPATCH_TYPE_X86_TSD
-MAPI_GLDISPATCH_ENTRY_FILES = $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_tsd.c
-MAPI_GLDISPATCH_ENTRY_FILES += $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_64_common.c
-MAPI_GLDISPATCH_ENTRY_FILES += $(top_srcdir)/$(MAPI_PREFIX)/entry_common.c
+MAPI_GLDISPATCH_ENTRY_FILES = mapi/entry_x86_tsd.c
+MAPI_GLDISPATCH_ENTRY_FILES += mapi/entry_x86_64_common.c
+MAPI_GLDISPATCH_ENTRY_FILES += mapi/entry_common.c
 endif
 
 if GLDISPATCH_TYPE_X86_64_TLS
-MAPI_GLDISPATCH_ENTRY_FILES = $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_64_tls.c
-MAPI_GLDISPATCH_ENTRY_FILES += $(top_srcdir)/$(MAPI_PREFIX)/entry_x86_64_common.c
-MAPI_GLDISPATCH_ENTRY_FILES += $(top_srcdir)/$(MAPI_PREFIX)/entry_common.c
+MAPI_GLDISPATCH_ENTRY_FILES = mapi/entry_x86_64_tls.c
+MAPI_GLDISPATCH_ENTRY_FILES += mapi/entry_x86_64_common.c
+MAPI_GLDISPATCH_ENTRY_FILES += mapi/entry_common.c
```

Comment:
This breaks 'make distcheck' for me (which, if I'm reading correctly, does a sanity build where srcdir != builddir):

> make[6]: Entering directory '/home/ajax/git/libglvnd/libglvnd-0.1.0/_build/sub/src/GLdispatch/vnd-glapi'
/bin/sh ../../../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I../../../../../src/GLdispatch/vnd-glapi -I../../..   -I../../../../../include -I../../../../../src/util -I../../../../../src/GLdispatch/vnd-glapi/mapi -I../../../src/GLdispatch/vnd-glapi/mapi -I../../../../../src/GLdispatch/vnd-glapi/mapi/glapi -I../../../../../ -DMAPI_ABI_HEADER=\"./glapi_mapi_tmp.h\"   -Wall -Werror -include config.h -fvisibility=hidden  -DUSE_X86_64_ASM -MT libglapi_la-g_glapi_inittable.lo -MD -MP -MF .deps/libglapi_la-g_glapi_inittable.Tpo -c -o libglapi_la-g_glapi_inittable.lo `test -f 'g_glapi_inittable.c' || echo '../../../../../src/GLdispatch/vnd-glapi/'`g_glapi_inittable.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../../../../../src/GLdispatch/vnd-glapi -I../../.. -I../../../../../include -I../../../../../src/util -I../../../../../src/GLdispatch/vnd-glapi/mapi -I../../../src/GLdispatch/vnd-glapi/mapi -I../../../../../src/GLdispatch/vnd-glapi/mapi/glapi -I../../../../../ -DMAPI_ABI_HEADER=\"./glapi_mapi_tmp.h\" -Wall -Werror -include config.h -fvisibility=hidden -DUSE_X86_64_ASM -MT libglapi_la-g_glapi_inittable.lo -MD -MP -MF .deps/libglapi_la-g_glapi_inittable.Tpo -c ../../../../../src/GLdispatch/vnd-glapi/g_glapi_inittable.c  -fPIC -DPIC -o .libs/libglapi_la-g_glapi_inittable.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../../../../../src/GLdispatch/vnd-glapi -I../../.. -I../../../../../include -I../../../../../src/util -I../../../../../src/GLdispatch/vnd-glapi/mapi -I../../../src/GLdispatch/vnd-glapi/mapi -I../../../../../src/GLdispatch/vnd-glapi/mapi/glapi -I../../../../../ -DMAPI_ABI_HEADER=\"./glapi_mapi_tmp.h\" -Wall -Werror -include config.h -fvisibility=hidden -DUSE_X86_64_ASM -MT libglapi_la-g_glapi_inittable.lo -MD -MP -MF .deps/libglapi_la-g_glapi_inittable.Tpo -c ../../../../../src/GLdispatch/vnd-glapi/g_glapi_inittable.c -o libglapi_la-g_glapi_inittable.o >/dev/null 2>&1
mv -f .deps/libglapi_la-g_glapi_inittable.Tpo .deps/libglapi_la-g_glapi_inittable.Plo
/bin/sh ../../../libtool  --tag=CC   --mode=link gcc  -Wall -Werror -include config.h -fvisibility=hidden  -DUSE_X86_64_ASM -no-undefined  -o libglapi.la  mapi/libglapi_la-entry_x86_64_tls.lo mapi/libglapi_la-entry_x86_64_common.lo mapi/libglapi_la-entry_common.lo mapi/libglapi_la-mapi_glapi.lo mapi/libglapi_la-stub.lo mapi/libglapi_la-table.lo mapi/libglapi_la-u_execmem.lo libglapi_la-g_glapi_inittable.lo mapi/libglapi_la-u_current_tls.lo  ../../../src/util/libutils_misc.la 
libtool:   error: 'mapi/libglapi_la-entry_x86_64_tls.lo' is not a valid libtool object

I think you want `$(srcdir)/mapi` instead of bare `mapi/`.